setup.py

```diff
@@ -94,4 +94,9 @@
     extras_require=extras_require,
     include_package_data=True,
     zip_safe=False,  # https://mypy.readthedocs.io/en/latest/installed_packages.html
+    entry_points={
+        "dask.array.backends": [
+            "cupy=dask.array.cupy_entry_point:CupyBackendEntrypoint"
+        ],
+    },
```

Comment:
This is specified both here (`setup.py`) and `setup.cfg`. I might be missing something but this seems redundant. Do we need this in both places?

Reply:
Right, I guess it's only needed in one of these files. Any idea which one is preferred?